### ISSUE TITLE
Paperspace support?

### ISSUE THREAD

**Issue by RONNYKHALIL** [TheLastBen/fast-stable-diffusion#2272]
Where did paperspace go? Kept getting this error and when I checked GitHub I noticed the paperspace link gone

Traceback (most recent call last):

  File /usr/local/lib/python3.9/dist-packages/IPython/core/interactiveshell.py:3378 in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)

  Cell In [2], line 9
    with open('/dev/null', 'w') as devnull:import requests, os, time, importlib;open('/notebooks/mainpaperspaceA1111.py', 'wb').write(requests.get('https://huggingface.co/datasets/TheLastBen/PPS/raw/main/Scripts/mainpaperspaceA1111.py').content);os.chdir('/notebooks');time.sleep(3);import mainpaperspaceA1111;importlib.reload(mainpaperspaceA1111);from mainpaperspaceA1111 import *;Deps(force_reinstall)

  File /notebooks/mainpaperspaceA1111.py:1
    Invalid username or password.
            ^
SyntaxError: invalid syntax

**Comment by TheLastBen** [comment]
I'm afraid you'll have to contact paperspace support for that

**Comment by RONNYKHALIL** [comment]
will do. but what about the your paperspace repository. why is it no longer available?
![Screenshot 2023-06-22 at 2 13 19 PM](https://github.com/TheLastBen/fast-stable-diffusion/assets/74345290/f5e6c8f9-ae8a-46d7-86d3-e0e41f77c811)

**Comment by TheLastBen** [comment]
@RONNYKHALIL that's up to paperspace to decide, I'm trying to contact them without an answer

**Comment by RONNYKHALIL** [comment]
Got ya, thanks!

**Comment by kerizane** [comment]
Could you provide this file?

https://huggingface.co/datasets/TheLastBen/PPS/raw/main/Scripts/mainpaperspaceA1111.py

**Comment by HelpANewbie** [comment]
Yeah Paperspace isn't letting us open the Stable Diffusion Notebook for some reason!

**Comment by RONNYKHALIL** [comment]
Heard back from Paperspace support: 
Thank you for contacting Paperspace support. We are currently working with Ben to resolve this issue. In the meantime, you can use this repo: https://github.com/gradient-ai/stable-diffusion. The main notebook in there has a Webui launcher that can be used.

**Comment by Headupdef** [comment]
> Heard back from Paperspace support: Thank you for contacting Paperspace support. We are currently working with Ben to resolve this issue. In the meantime, you can use this repo: https://github.com/gradient-ai/stable-diffusion. The main notebook in there has a Webui launcher that can be used.
> 
> hope they sort it out, Ben's repo is the best one out there by leaps and bounds, like night and day compared to any other I've used on paperspace

that repo sucks and is confusing, I know if they don't get this working they're losing a good customer here

**Comment by RONNYKHALIL** [comment]
💯 agree. already getting back into RunDIffusion jic

**Comment by Headupdef** [comment]
All support told me was this....

There is wider problem that we are aware of and are working to resolve.

Best regards,

**Comment by AdrianGomezCL** [comment]
Same problem here, if PaperSpace dont fix this im going to change the provider.

**Comment by Headupdef** [comment]
I also responded to the email with the same thing lol

**Comment by banMo88** [comment]
same question.Let me know when it's fix,thanks :)

**Comment by SadSAYA** [comment]
same error
Will Paperspace also ban SD?

**Comment by toyssamurai** [comment]
I switched to use Voldemort for the time being.

**Comment by Archon332** [comment]
Same here.

**Comment by MherNetDev** [comment]
Any news? :(

**Comment by Headupdef** [comment]
Well after I emailed back that I will have to cancel my acct if this
doesn't get back up and running all they said was
We understand. Again, we are doing what we can to resolve the issue and
will update you accordingly.

Best regards,

**Comment by TheLastBen** [comment]
It's back up and running

**Comment by banMo88** [comment]
> 

you are the bast Ben!!!

**Comment by banMo88** [comment]
I just finished and planned to share my own temporary notebook literally just 5 mins before you announce the problem had been solved😂. But no matter what, it's glad to see your notebook is back.

**Comment by MherNetDev** [comment]
Tysm Ben🔥🙌🙌

**Comment by RONNYKHALIL** [comment]
yipeeee

**Comment by Headupdef** [comment]
Thanks Ben!

**Comment by Gamemaster2930** [comment]
Good job Ben. Thank you!
